dts/bindings/mtd/microchip,xec-eeprom.yaml

```diff
@@ -0,0 +1,38 @@
+# Copyright (c) 2022 Microchip Technology Inc.
+# SPDX-License-Identifier: Apache-2.0
+
+description: Microchip on-chip EEPROM
+
+compatible: "microchip,xec-eeprom"
+
+include: [eeprom-base.yaml, pinctrl-device.yaml]
+
+properties:
+    reg:
+      required: true
+
+    girqs:
+      type: array
+      required: true
+      description: |
+        Array of GIRQ and bit position pairs for each interrupt
+        signal the block generates.
+
+    pcrs:
+      type: array
+      required: true
+      description: PS2 PCR register index and bit position
+
+    pinctrl-0:
+      required: true
+
+    pinctrl-names:
+      required: true
```

Comment:
If pinctrl is not supported by all SoCs where this driver is supported, these devicetree properties must be made optional.

Reply:
properties made optional